pkgs/by-name/ve/vectorcode/package.nix

```diff
@@ -45,7 +46,51 @@ let
         dependencies = old.dependencies ++ [
           self.chroma-hnswlib
         ];
-        doCheck = false;
+
+        pytestFlags = [
+          # This package sometimes has issues running multiple tests in parallel.
+          # ValueError: An instance of Chroma already exists for ephemeral with different settings
+          "--numprocesses=1"
+        ];
```

Comment:
Is this relevant to the base package too? Should I move this to [pkgs/development/python-modules/chromadb/default.nix](https://github.com/NixOS/nixpkgs/blob/9541941f0405c4d1145923a436b205458d0b4d2a/pkgs/development/python-modules/chromadb/default.nix) ?

Reply:
Please remove this and disable the affected test instead. No other package in nixpkgs resorts to this flag.